lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -118,6 +119,8 @@ private void init(@Nullable AttributeSet attrs) {
       addColorFilter(new SimpleColorFilter(ta.getColor(
           R.styleable.LottieAnimationView_lottie_colorFilter, Color.TRANSPARENT)));
     }
+
+    scale = ta.getFloat(R.styleable.LottieAnimationView_lottie_scale, Float.MAX_VALUE);
```

Comment:
How about you just call setScale on `lottieDrawable` here and if the attribute doesn't exist, then set it to the logic that's currently below and removing it there. Since you solved the issue of the drawable overwriting the scale on setComposition, you can set it here and be done.